hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/clustering/run/strategy/MultipleSparkJobExecutionStrategy.java

```diff
@@ -116,7 +116,7 @@ public HoodieWriteMetadata<HoodieData<WriteStatus>> performClustering(final Hood
       Stream<HoodieData<WriteStatus>> writeStatusesStream = FutureUtils.allOf(
               clusteringPlan.getInputGroups().stream()
                   .map(inputGroup -> {
-                    if (getWriteConfig().getBooleanOrDefault("hoodie.datasource.write.row.writer.enable", false)) {
```

Comment:
@yihua : this was intentionally kept it as false (default value). 
So only if user explicitly enabled row writer, we will enable row writer w/ clustering.